infoBot.js

```diff
@@ -105,27 +111,28 @@ client.on("message", async msg => {
 
     //gets the command in lower case
     const commandName = args.shift().toLowerCase();
-    
+
     // get the command or its aka
     const command = client.commands.get(commandName) || client.commands.find(cmd => cmd.aka && cmd.aka.includes(commandName));
 
     // if no command dont do anything
     if (!command) return;
-    
+
     // disallows commands in dm's to run as commands in dms if it is set to guild only
-    if (command.guildOnly && msg.channel.type !== 'text') {
+    if (command.guildOnly && msg.channel.type !== 'text') 
+    {
         return msg.reply('Sorry - I can\'t do that in a DM');
     }
-    
+
     // only runs if below Guild id's (EXP = `260843215836545025`) 762249085268656178 is testing server
     if (command.guildOnly && (guild != `762249085268656178` && guild != `260843215836545025`)) {
         console.log(`Not correct guild`);
```

Comment:
I think we should add what guild this IS in, not sure if it will error out if in DM though... so maybe

```suggestion
        if (!guild) 
        {
            console.log(`Not correct Guild - Seems to be a DM`);
        }
            else if (guild != `762249085268656178` && guild != `260843215836545025`) 
       {
            console.log('Not correct guild - Guild id {$guild};
       }
       else {console.log('Not correct guild`);}
```

Reply:
this logic wont work but ill add the guild into ( either testing or exp) into the bot log

Reply:
Not sure why that logic won't work, but if it won't the whole point is we want to have it report if someone is trying to use the bot outside of the allowed locations. It's not a big deal if they're using it inside the two servers. Basically if somebody gets a this server isn't allowed reply we want it logged where they try to use it from as a log item.

Reply:
okey ill do that